config/swagger.yaml

```diff
@@ -37,6 +37,22 @@ paths:
 ## External APIs definitions
 ###
 
+  /top:
```

Comment:
I'm reintroducing `/top` endpoint for full backward compatibility. It'll be removed with other legacy endpoints

Reply:
@jur0 @velzevur ^

Reply:
I see @jur0 had fixed the python UATs, do we still need it?